target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-lenovo-ix2-common.dtsi

```diff
@@ -0,0 +1,221 @@
+/* 
+ * Common Device tree file for the Lenovo / Iomega IX2 Devices
+ *
+ * Copyright (c) 2015-2016 bodhi <mibodhi@gmail.com>
+ * Copyright (c) 2015      balbes150
+ *
+ * This file is licensed under the terms of the GNU General Public
+ * License version 2.  This program is licensed "as is" without any
+ * warranty of any kind, whether express or implied.
+ */
+
+/dts-v1/;
```

Comment:
dts-v1 is only needed once. Please look where others put it (kirdwood.dtsi, other dtsi, or dts).